README.rst

```diff
@@ -1086,3 +1086,83 @@ And more...
 -----------
 
 Additional code samples are available in the `test directory <https://github.com/irods/python-irodsclient/tree/master/irods/test>`_
+
+
+=======
+Testing
+=======
+
+Setting up and runnning tests
+-----------------------------
+
+The Python iRODS Client comes with its own suite of tests.  Some amount of setting up may be necessary first:
+
+    1. Use :code:iinit to specify the iRODS client environment.
+       For best results, point the client at a server running on the local host.
+
+    2. Install the python-irodsclient along with the :code:`unittest unittest_xml_reporting` module or the older :code:`xmlrunner` equivalent.
+          - for PRC versions 1.1.1 and later: :code:`pip install ./path-to-python-irodsclient-repo[tests]`  (when using a local Git repo)
+                                          or: :code:`pip install python-irods[tests]`  (when installing directly from PyPI)
```

Comment:
see how these render at github by looking at your branch's README.   i suggest doing...

later:
 :code:
or:
 :code:

to break up the words from the code itself, easier to read.